### PR TITLE
Refactored Android Library and App

### PR DESCRIPTION
**Pull request by ReeceHughes**
Refactored Android Library and App, removing all Google names and replacing them with Flume.live. 
This library is now available on Maven and JCenter

Other Note:
- Remove Log lines for release

### CODE REVIEW

File: mobile/android/WirelessDebugger/wirelessdebugger/build.gradle [hunk 1]
```diff
@@ -19,19 +26,38 @@ android {
...
+
+publish {
+    groupId = 'live.flume.wireless.debugger'
+    artifactId = 'wirelessdebugger'
+    repoName = 'wireless-debugger'
+    publishVersion = '1.0.0'
+    desc = 'Wireless Debugging Library for Android - Initial Release'
+    licences = ['Apache-2.0']
+    uploadName='WirelessDebugger'
+    website = 'https://github.com/sumnerevans/wireless-debugging '
```
Comment (cnavrides) [inline mobile/android/WirelessDebugger/wirelessdebugger/build.gradle]: nitt: Have extra space here

File: mobile/android/WirelessDebugger/wirelessdebugger/src/main/java/live/flume/wireless/debugger/LogReader.java [hunk 2]
```diff
@@ -64,7 +64,7 @@ public void run() {
 
             String logLine = "";
 
-            Log.d(TAG, "Begin Read line in buffer");
+            // Log.d(TAG, "Begin Read line in buffer");
```
Comment (cnavrides) [inline mobile/android/WirelessDebugger/wirelessdebugger/src/main/java/live/flume/wireless/debugger/LogReader.java]: Remove comment please

File: mobile/android/WirelessDebugger/wirelessdebugger/src/main/java/live/flume/wireless/debugger/WebSocketMessenger.java [hunk 3]
```diff
@@ -36,7 +36,7 @@
     public static WebSocketMessenger buildNewConnection(String socketAddress, String apiKey,
                                                         String appName) {
         URI uri;
-        Log.i(TAG, "URI: " + socketAddress);
+        // Log.i(TAG, "URI: " + socketAddress);
```
Comment (cnavrides) [inline mobile/android/WirelessDebugger/wirelessdebugger/src/main/java/live/flume/wireless/debugger/WebSocketMessenger.java]: remove this line please

File: mobile/android/WirelessDebugger/wirelessdebugger/src/main/java/live/flume/wireless/debugger/WebSocketMessenger.java [hunk 4]
```diff
@@ -67,7 +67,7 @@ private WebSocketMessenger(URI uri, String apiKey, String appName) {
      */
     @Override
     public void onOpen(ServerHandshake serverHandshake) {
-        Log.i(TAG, "Connection opened!");
+        //Log.i(TAG, "Connection opened!");
```
Comment (cnavrides) [inline mobile/android/WirelessDebugger/wirelessdebugger/src/main/java/live/flume/wireless/debugger/WebSocketMessenger.java]: Here and below

File: mobile/android/WirelessDebugger/wirelessdebugger/build.gradle (hunk 1 above)
Comment (ReeceHughes) [inline mobile/android/WirelessDebugger/wirelessdebugger/build.gradle]: Where exactly?

Comment (sumnerevans) [inline mobile/android/WirelessDebugger/wirelessdebugger/build.gradle]: after the `g` before the last `'`

Comment (ReeceHughes) [inline mobile/android/WirelessDebugger/wirelessdebugger/build.gradle]: Nevermind, at the end of website.

**Pull request merged**